src/data/markdown/docs/02 javascript api/04 k6-data/1-SharedArray.md

```diff
@@ -28,7 +28,7 @@ Which means that for the most part if you currently have an array data structure
 import { SharedArray } from 'k6/data';
 
 const data = new SharedArray('some name', function () {
-  // here you can open files, and then do additional processing on them or just generate the data dynamically
+  // it's important to open big files (generate a big amount of data) inside this function
```

Comment:
```suggestion
  // here you can generate the data dynamically or open files and do additional processing
  // it's important to open large files inside this function to avoid memory saturation
```